providers/azureblob/src/main/java/org/jclouds/azure/storage/filters/SharedKeyLiteAuthentication.java

```diff
@@ -203,6 +240,24 @@ public String createStringToSign(HttpRequest request) {
       return buffer.toString();
    }
 
+   private void appendPayloadMetadataForSharedKey(HttpRequest request, StringBuilder buffer) {
```

Comment:
Probably we can just early return if the request payload is null, and don't check it every time?

Reply:
done